### PR TITLE
Feat/UI styling

### LINKED ISSUE #27: Remove "show me the facts" button from book of cat

### PR DESCRIPTION
**Pull request by jsahim**
Change Summary: Removed Show Facts Button, added functionality for random color background and catType display. Also added new fonts and styling.
Functions: getRandomColor(), getCatPath ()

Close #27

### CODE REVIEW

File: src/components/Finale/Finale.js
Comment (Jesusborjas006) [inline src/components/Finale/Finale.js:1]: Great job displaying a different cat!

**Review by Jesusborjas006 — APPROVED** [review]

**Review by BriBourassa — APPROVED** [review]
Your design skills never cease to amaze me Jeff!!

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&family=VT323&display=swap" rel="stylesheet">
```
Comment (qrispi) [inline public/index.html:6]: This font is really cool! You can also import fonts directly into the CSS file with React!

File: src/components/App/App.js [hunk 2]
```diff
@@ -4,9 +4,16 @@ import Story from "../Story/Story";
 import Book from "../Book/Book";
 import { Route, Switch, Redirect } from "react-router-dom";
 
+
 function App() {
+
+  const getRandomColor = () => {
+    const colors = ["#d71377", "#0b97a8", "#522d94", "#90c62a", "#fcd00b"]
+    return colors[Math.floor((Math.random() * 5))]
+  }
```
Comment (qrispi) [inline src/components/App/App.js:13]: This looks super cool!

File: src/components/Finale/Finale.js [hunk 3]
```diff
@@ -2,7 +2,18 @@ import { useSelector } from "react-redux";
...
+  const catMorality = useSelector((state) => state.cat.morality);  
+
+
+  const getCatPath = () => {
+    if (catMorality > 0) {
+      return require('../../images/good-cat-fly.gif')
+    } else if (catMorality === 0) {
+      return require('../../images/neutral-cat.gif')
+    } else {
+      return require('../../images/evil-cat.gif')
```
Comment (qrispi) [inline src/components/Finale/Finale.js:14]: Love the kitties!

File: src/images/frame-backdrop.png
Comment (qrispi) [inline src/images/frame-backdrop.png:1]: Smart fix!

**Review by qrispi — APPROVED** [review]
Jeff this awesome! It actually looks like a game now!

**Pull request merged**